geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -1,15 +1,23 @@
-{% load i18n %}
+{% load i18n corpus_extras %}
 <section id="iiif-viewer">
     <h1 class="sr-only">
         {# Translators: accessibility label for document images and transcription section #}
         {% translate "Images and transcription" %}
     </h1>
     {% if document.has_image %}
-        <ol class="iiif-images">
-            {% for image in document.iiif_images %}
-                <li id="image-{{ forloop.counter }}">{{ image|safe }}</li>
-            {% endfor %}
-        </ol>
+        <noscript>
+            <ol id="iiif-images-fallback">
+                {% for image_dict in document.iiif_images %}
+                    <li id="image-{{ forloop.counter }}">
+                        <a href="{{ image_dict.image|iiif_image:"size:width=2880" }}">
+                            <img src="{{ image_dict.image|iiif_image:"size:width=500" }}" alt="{{ image_dict.label }}" title="{{ image_dict.label }}" loading="lazy">
+                        </a>
+                    </li>
+                {% endfor %}
+            </ol>
+        </noscript>
+        <div id="iiif-images" data-iiif-urls="{{ document.iiif_image_ids|iiif_info_json }}">
```

Comment:
if we reference the piffle image objects, we could use the `info()` method

(I don't know if we have easy access to them here; this seems clunky to me but I think acceptable)

Reply:
Done in 9b3b1d6, much simpler, thanks!